protocol-units/da-sequencer/client/src/lib.rs

```diff
@@ -76,5 +77,5 @@ impl DaSequencerClient {
 }
 
 pub fn sign_batch(batch_data: &[u8], signing_key: &SigningKey) -> Signature {
-        signing_key.sign(batch_data)
+	signing_key.sign(batch_data)
```

Comment:
Meaningless fmt diff, Check your default IDE formatter settings. They are not applying the rules set in rustfmt.toml